docs/use_tables.md

```diff
@@ -0,0 +1,11 @@
+# Table explorer
```

Comment:
Leave it out and create a story to develop the missing parts.

Reply:
added a few more contents